### PR TITLE
Added redirection from root to login if no auth token

### LINKED ISSUE #109: Redirect users without Auth-Token to login page
2 Cases:
- When the user arrives at an endpoint that requires authentication
- After the user is on the page that requires authentication but receives a 401 from the server from subsequent AJAX requests (due to token expiring or revoked)

### PR DESCRIPTION
**Pull request by Ommy**
Resolves #109

### CODE REVIEW

File: src-frontend/app/routes/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+    beforeModel: function() {
+        if (!Cookies.get('authorizationToken')) {
+            this.transitionTo('login');
+        }
+    }
+})
```
Comment (Kapin) [inline src-frontend/app/routes/index.js]: newline plz

File: src-frontend/app/controllers/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+    beforeModel: function() {
+        debugger
+        if (Cookies.get('authorizationToken')) {
+            console.log("Has token");
+        }
+    }
+});
```
Comment (Kapin) [inline src-frontend/app/controllers/index.js]: newline plz

File: src-frontend/app/controllers/index.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+});
```
Comment (Trinovantes) [inline src-frontend/app/controllers/index.js]: new line

File: src-frontend/app/routes/application.js [hunk 4]
```diff
@@ -15,5 +15,13 @@ export default Ember.Route.extend({
                 parentView: 'application'
             });
         }
+    },
+    actions: {
```
Comment (Trinovantes) [inline src-frontend/app/routes/application.js]: is this supposed to be a new property?

Comment (Ommy) [inline src-frontend/app/routes/application.js]: That's the ember's actions. It catches events like error on request

File: src-frontend/app/routes/application.js [hunk 5]
```diff
@@ -15,5 +15,13 @@ export default Ember.Route.extend({
...
+        error: function(error, transition) {
+            // If unauthorized
```
Comment (Trinovantes) [inline src-frontend/app/routes/application.js]: unnecessary comment

File: src-frontend/app/routes/index.js [hunk 6]
```diff
@@ -0,0 +1,4 @@
+import Application from '../routes/application';
```
Comment (ConnorCimowsky) [inline src-frontend/app/routes/index.js]: Can we do `src-frontend/routes/application` instead? In all of the files :innocent:

File: src-frontend/app/routes/procedures.js [hunk 7]
```diff
@@ -1,4 +1,4 @@
-import Ember from 'ember';
+import Application from '../routes/application';
 
 export default Ember.Route.extend({
```
Comment (Trinovantes) [inline src-frontend/app/routes/procedures.js]: missed extend

**Comment by Trinovantes** [comment]
+1

**Comment by Ommy** [comment]
"Looks ok to me" - @Kapin

**Pull request merged**